### DISCUSSION TITLE
Is dit niet hetzelfde als een Either type?

### DISCUSSION THREAD

**Discussion by robbertvdzon-alliander** [alliander-opensource/Result#3] [Q&A] [answered]
Hoi Daan!

Ik zag een aanpassing op de full stack java gilde met een link naar deze repo, dus vandaar dat ik daar even naar keek.
Maar is dit Result type niet precies wat een Either type is in bijvoorbeeld vavr of arrow? 

https://arrow-kt.io/docs/apidocs/arrow-core-data/arrow.core/-either/
https://docs.vavr.io/#_either

**Comment by dvberkel** [comment, marked as answer]
Hi Robbert,

Dit is inderdaad net al s het `Either` type van bijvoorbeeld arrow. We hebben echter andere eisen aan dat type, die onder andere uitgelegd zullen worden bij de meeting van het Java gilde
